### PR TITLE
Small fixes to make the script work with the correct file names;

### PR DESCRIPTION
**Pull request by prostyman**
Don't merge it yet, let's talk about it first so I can show you how the pull requests work.

Also I added you (@Lunetta ) as a code reviewer, so complain if you don't like some parts (for example you should tell me to remove the debug code instead of putting a Todo comment)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
-/venv
-/.idea
\ No newline at end of file
+venv/
+.idea/
+output_configs/
```
Comment (Lunetta) [inline .gitignore:3]: what does diviete icon means? 
what is /.idea?
is outp_configs your folder to save results?

File: MergeDatiJinja.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from jinja2 import Template
-with open('ConfTemplate.txt','r') as fp:
```
Comment (Lunetta) [inline MergeDatiJinja.py:2]: why did you rename this file? As you have done this script doesn't work. Should you maybe change the line about read file?

File: create_config.py [hunk 3]
```diff
@@ -1,21 +1,24 @@
-from jinja2 import Template
```
Comment (Lunetta) [inline create_config.py:1]: really? O.O

File: create_config.py [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
+import os
 import json
+from jinja2 import Template
 
+DEBUG = True
+output_dir = "output_configs"
```
Comment (Lunetta) [inline create_config.py:5]: The configuration files have to be in a specific folder, I can't create a new one.

File: create_config.py [hunk 5]
```diff
@@ -1,21 +1,24 @@
...
+    loadtemplate = fp.read()
 
-with open('ConfTemplate.txt','r') as fp:
-    loadtemplate=fp.read()
-
-with open('Clients.txt') as json_file:
-    clients=json.load(json_file)
-
-jinjatemplate=Template(loadtemplate)
+with open('Clients.json', 'r') as json_file:
```
Comment (Lunetta) [inline create_config.py]: did it work?

File: create_config.py [hunk 6]
```diff
@@ -1,21 +1,24 @@
...
 
+jinjatemplate = Template(loadtemplate)
 
-for key in clients:
-    hostname=key
-    ip=clients[key]
-    conf=jinjatemplate.render(hostname=hostname,ip=ip)
-myfile=open(key+'.conf','w') #to move in the for loop
-result=myfile.writelines(conf)#to move in the for loop
+os.makedirs(output_dir, exist_ok=True)
```
Comment (Lunetta) [inline create_config.py:15]: ok but I prefer as I did. I am not sure about to create a new folder because after that you have run the script you have to unpack all files and delate the folder

File: .gitignore (hunk 1 above)
Comment (prostyman) [reply .gitignore:3]: In .gitignore '/' at the end of a word indicates that it is a directory to ignore;
.idea/ is a directory PyCharm creates with all the information about the project. I added it here to avoid committing useless files, but if you don't like it I can remove it and decide locally what to ignore; 
Yes, output_config is a folder to put the output. I put it here so we avoid committing the output of the script

File: MergeDatiJinja.py (hunk 2 above)
Comment (prostyman) [reply MergeDatiJinja.py:2]: I changed it just to make it clear that it is a Jinja2 template.
It is working for me, the extension of the file shouldn't matter when parsing

File: create_config.py (hunk 4 above)
Comment (prostyman) [reply create_config.py:5]: Ok, I will change this then. Is it the same folder of the clients json file?

If you are ok with that, I would add an optional cmd line argument. The default value would be the same folder as the script, but if you specify a folder it will copy the output there. (It is handy for debugging, so we don't create necessarily a lot of files in the code folder)

File: create_config.py (hunk 5 above)
Comment (prostyman) [reply create_config.py]: It is working for me, I will double check tonight if I find 5 minutes. If you want you can also checkout the pretty_changes branch and try it on your machine

File: create_config.py (hunk 6 above)
Comment (prostyman) [reply create_config.py:15]: See my comment for line 6, we can change it if you prefer
